FV3/wrapper/templates/_wrapper.pyx

```diff
@@ -62,16 +63,18 @@ cdef extern:
 {% endfor %}
 
 cdef get_quantity_factory():
-    cdef int nx, ny, nz, nz_soil
+    cdef int nx, ny, nz, nz_soil, n_topo_variables
     get_centered_grid_dimensions(&nx, &ny, &nz)
     get_nz_soil_subroutine(&nz_soil)
+    get_n_topo_variables_subroutine(&n_topo_variables)
```

Comment:
The subroutine name called here isn't consistent with the imported name (`get_n_topographic_variables`).